core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConfig.java

```diff
@@ -152,6 +152,16 @@
     @ConfigItem(defaultValue = "all")
     TestType type;
 
+    /**
+     * If a class matches this pattern then it will be cloned into the Quarkus ClassLoader even if it
+     * is in a parent first artifact.
+     *
+     * This is important for collections which can contain objects from the Quarkus ClassLoader, but for
+     * most parent first classes it will just cause problems.
+     */
+    @ConfigItem(defaultValue = "java.*")
```

Comment:
@stuartwdouglas sorry to raise this question so late but shouldn't it be `java\..*`? Because with this, you also catch the Bean Validation classes starting with `javax.` for instance and anything starting with `java` at large.

Reply:
good catch.